microservices/dms_sync/providers/dealervault/parser.py

```
@@ -0,0 +1,372 @@
+        make = record.get("Make")
+        model = record.get("Model")
+        year = self.safe_int(record.get("Year"))
+        color = record.get("Exterior Color")
+        
+        # New/Used: "New/Used" char col 'N'/'U'
+        nu_val = record.get("New/Used")
+        new_used = "New" if nu_val == "N" else "Used" if nu_val == "U" else None
+
+        # --- Service Info ---
+        ro_number = str(record.get("RO Number") or "")
+        open_date = self.parse_date(record.get("Open Date"))
```

Ruff (0.14.14)

[error] Local variable `open_date` is assigned to but never used

Remove assignment to unused variable `open_date`

(F841)